hadoop-ozone/dist/src/main/smoketest/s3/commonawslib.robot

```diff
@@ -39,18 +39,6 @@ Execute AWSS3Cli
     ${output} =       Execute                     aws s3 --endpoint-url ${ENDPOINT_URL} ${command}
     [return]          ${output}
 
-Install aws cli
```

Comment:
Thanks @adoroszlai  for the improvement. That will save us unnecessary CI execution time. 
I have one question: can we check and install only if the base image does not have it in the acceptance test? This way, the test itself is still self-contained without depending on specific docker images.

Reply:
Thanks @xiaoyuyao for the review.  The installation logic is idempotent (`yum` does nothing if the package is already installed), so we could simply keep it without any new check.  I have considered this option, but we already have other tools which are not installed during tests, rather assumed to be present in the environment (eg. `boto3` most recently).

Reply:
Thanks @adoroszlai  for the explanation. Can keep the s3 install as-is and add those idempotent installation (like boto3) to the test to make it more resilient? This can be done in follow up JIRAs.